Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM maven:3.9.6 AS build
+
+RUN mkdir /code
```

Comment:
You can remove the directive `RUN mkdir /code` and move the `WORKDIR /code`  to line 3 instead